openedx/stanford/djangoapps/register_cme/tests/test_form.py

```diff
@@ -151,3 +151,31 @@ def test_export_countries(self, country):
         form = ExtraInfoForm(data)
         is_valid = form.is_valid()
         self.assertFalse(is_valid)
+
+    @data(None, 'ab', 'longsunet')
+    def test_clean_affiliation_sunet_id(self, sunet_id):
+        data = self.data
+        data['affiliation'] = 'Stanford University'
+        data['stanford_department'] = 'Medicine'
+        data['sunet_id'] = sunet_id
+        form = ExtraInfoForm(data)
+        is_valid = form.is_valid()
+        self.assertFalse(is_valid)
+
+    @data(None, 'invalid_dept')
```

Comment:
It's generally beneficial to avoid abbreviations (clarity over brevity for maintainability),
so we can just call this `invalid_department`.